src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,2 @@
+package core.basesyntax;public class Square {
```

Comment:
The class declaration should not begin with an empty line. Please remove the empty line to adhere to the coding standards.